proto/acl_counting.proto

```diff
@@ -4,15 +4,16 @@
 syntax = "proto3";
 
 /*
- * ACL counts for devices
+ * Rule counts of devices
```

Comment:
nit: Rule counts *for* devices.  "of" implies that the keys are Rules and the counts of number of devices.

Reply:
Done